www/pages/web-create-project.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: page
+permalink: /web-create-project
+title: Creating projects and uploading files
+---
+
+There are several methods of creating a new project in Mergin:
+
+## Create a blank project
+
+- To create a new project, select **+ Create** in front of **Projects**
+- A new window will appear:
+  - For **Name** type your project name
+  - For **Project owner** select your username or your organisation
+  - By default the project is private. You can choose the option to make it **Public**
```

Comment:
Maybe add a screenshot with 1. Create button highlighted on dashboard, 2. create project window (with something typed in)  ... we should make sure nobody gets stuck there :-)